tsconfig-strict.json

```diff
@@ -714,15 +587,37 @@
     "extensions/objects/templates/svg-editor.constants.ts",
     "extensions/objects/templates/svg-editor.constants.ajs.ts",
     "extensions/rich_text_components/Collapsible/directives/oppia-noninteractive-collapsible.component.ts",
-    "extensions/rich_text_components/Link/directives/oppia-noninteractive-link.component.ts",
-    "extensions/rich_text_components/Link/directives/oppia-noninteractive-link.component.spec.ts",
```

Comment:
@EeshaArif Please note that replacing these files with `"extensions/rich_text_components/Link/*.ts",` won't work because the directores inside these folder are not covered, to cover these you'll have to introduce `extensions/rich_text_components/Link/directives/*.ts` this file path. Can you please go through current and this PR https://github.com/oppia/oppia/pull/13707 to prevent such cases ?

Reply:
done, covered in #13707